### PR TITLE
docs: response time histogram worst-in-input and all option

### PR DESCRIPTION
**Pull request by rokamu623**
## Description
Added code and images for the `all` and `worst-in-input` options of the response time histogram.
Changed the image to be the same as the stack bar image.
Erase unusing option (`best-to-worst`).

## Related links
Jira: https://tier4.atlassian.net/browse/RT2-862

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: docs/visualization/path/response_time.md [hunk 1]
```diff
@@ -35,30 +35,41 @@ The following scripts generate histograms of response time. Horizontal axis of t
...
 # plot worst case
 plot = Plot.create_response_time_histogram_plot(path, case='worst')
 plot.show()
 ```
 
-![response_time_worst_histogram](../../imgs/response_time_worst_histogram.png)
+![response_time_worst_histogram](../imgs/response_time_histogram_worst.png)
+
+```python
+# plot worst case
```
Comment (nabetetsu) [inline docs/visualization/path/response_time.md:67]:
```suggestion
# plot worst-in-input case
```

File: docs/visualization/path/response_time.md [hunk 2]
```diff
@@ -35,30 +35,41 @@ The following scripts generate histograms of response time. Horizontal axis of t
 # plot best-to-worst case
 plot = Plot.create_response_time_histogram_plot(path)
 plot.show()
+# or
+# plot = Plot.create_response_time_timeseries_plot(path, case='all')
```
[the remark this answers is not in the document]
Comment (rokamu623) [reply docs/visualization/path/response_time.md]: Fixed it in [416a00e](https://github.com/tier4/CARET_doc/pull/241/commits/416a00ebc784c5fe387b2a325711ebfeac899937).

File: docs/visualization/path/response_time.md (hunk 1 above)
Comment (rokamu623) [reply docs/visualization/path/response_time.md]: @nabetetsu 
Fixed it in [416a00e](https://github.com/tier4/CARET_doc/pull/241/commits/416a00ebc784c5fe387b2a325711ebfeac899937).

**Review by nabetetsu — APPROVED** [review]
LGTM :+1:

**Comment by rokamu623** [comment]
If one or more Emb teams approve, merge.

**Review by bopeng-saitama — APPROVED** [review]
LGTM

**Pull request merged**